src/daemon.rs

```diff
@@ -192,6 +213,12 @@ impl Checker for Daemon {
 
         self.timeout_battery = time::Duration::from_millis(self.settings.delay_battery);
         self.timeout = time::Duration::from_millis(self.settings.delay);
+
+        if let Some(name) = &self.settings.csv_file {
+            if !Path::new(name).exists() {
+                File::create(name).expect("CSV file could not be created.");
+            }
+        }
```

Comment:
Maybe we should have an option to just override it on startup so u dont have to go in and delete the previous one every time

Reply:
New folders for each day? We could create a new CSV file hourly. We don't want massive individual files.

Reply:
yep sounds good, we can also just use the timestamp as the file name

Reply:
Good point about the size limit. It just writes out the current cpu objects with the to_csv to format them as strings then write it to a file. It does not store the whole thing in memory. I think a size limit would be very good so it does not break a hard drive.

Reply:
The file was getting multiple kilobytes in a few seconds

Reply:
yea, also the size limits are acutally all in mebibytes instead of megabites

Reply:
yea